opendbc/car/byd/carstate.py

```diff
@@ -0,0 +1,145 @@
+from opendbc.car import Bus, structs
+from opendbc.can.parser import CANParser
+from opendbc.can.can_define import CANDefine
+from opendbc.car.common.conversions import Conversions as CV
+from opendbc.car.interfaces import CarStateBase
+from opendbc.car.byd.values import DBC, CANBUS
+
+# Multiplier between GPS ground speed to the meter cluster's displayed speed
+HUD_MULTIPLIER = 1.068
+
+class CarState(CarStateBase):
+  def __init__(self, CP):
+    super().__init__(CP)
+    print(CP.carFingerprint)
+    can_define = CANDefine(DBC[CP.carFingerprint][Bus.pt])
+
+    self.shifter_values = can_define.dv["DRIVE_STATE"]['GEAR']
+    self.set_distance_values = can_define.dv['ACC_HUD_ADAS']['SET_DISTANCE']
+
+    self.prev_angle = 0
+    self.lss_state = 0
+    self.lss_alert = 0
+    self.tsr = 0
+    self.ahb = 0
+    self.passthrough = 0
+    self.lka_on = 0
+    self.HMA = 0
+    self.lkas_rdy_btn = False
+    self.lkas_faulted = False
+
+  def update(self, can_parsers) -> structs.CarState:
+    cp = can_parsers[Bus.pt]
+    cp_cam = can_parsers[Bus.cam]
+    ret = structs.CarState()
+
+    self.tsr = cp_cam.vl["LKAS_HUD_ADAS"]['TSR']
+    self.lka_on = cp_cam.vl["LKAS_HUD_ADAS"]['STEER_ACTIVE_ACTIVE_LOW']
+    self.lkas_rdy_btn = cp.vl["PCM_BUTTONS"]['LKAS_ON_BTN']
+    self.abh = cp_cam.vl["LKAS_HUD_ADAS"]['SET_ME_XFF']
+    self.passthrough = cp_cam.vl["LKAS_HUD_ADAS"]['SET_ME_X5F']
+    self.HMA = cp_cam.vl["LKAS_HUD_ADAS"]['HMA']
+    self.lkas_healthy = cp_cam.vl["STEERING_MODULE_ADAS"]['EPS_OK']
```

Comment:
What is this for? Just set ret.steerFaultTemporary. We shouldn't need special handling